client/store/index.js

```diff
@@ -3,12 +3,42 @@ import {createLogger} from 'redux-logger'
 import thunkMiddleware from 'redux-thunk'
 import {composeWithDevTools} from 'redux-devtools-extension'
 import auth from './auth'
+import axios from 'axios'
```

Comment:
looks like you accidentally pasted code from your /store/productsReducer.js file here... lines 6 to 16, lines 26 to 39 and line 44 can be deleted... I think the index.js file can be (so only changing lines 6, 8 and 16:

import {createStore, combineReducers, applyMiddleware} from 'redux'
import {createLogger} from 'redux-logger'
import thunkMiddleware from 'redux-thunk'
import {composeWithDevTools} from 'redux-devtools-extension'
import auth from './auth'
import products from './products'; // pending you change productsReducer to products

const reducer = combineReducers({ auth, products }) // combine the reducer you created to all other reducers in the project
const middleware = composeWithDevTools(
  applyMiddleware(thunkMiddleware, createLogger({collapsed: true}))
)
const store = createStore(reducer, middleware)

export default store
export * from './auth'
export * from './products' // export all